securitycenter/notificationsv2/notifications_test.go

```diff
@@ -0,0 +1,265 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package notificationsv2
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	securitycenter "cloud.google.com/go/securitycenter/apiv2"
+	"cloud.google.com/go/securitycenter/apiv2/securitycenterpb"
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+	"github.com/google/uuid"
+)
+
+func orgID(t *testing.T) string {
+	orgID := os.Getenv("GCLOUD_ORGANIZATION")
+	if orgID == "" {
+		t.Skip("GCLOUD_ORGANIZATION not set")
+	}
+	return orgID
+}
+
+func projectID(t *testing.T) string {
+	projectID := os.Getenv("SCC_PUBSUB_PROJECT")
+	if projectID == "" {
+		t.Skip("SCC_PUBSUB_PROJECT not set")
+	}
+	return projectID
+}
+
+func pubsubTopic(t *testing.T) string {
+	pubsubTopic := os.Getenv("SCC_PUBSUB_TOPIC")
+	if pubsubTopic == "" {
+		t.Skip("SCC_PUBSUB_TOPIC not set")
+	}
+	return pubsubTopic
+}
+
+func pubsubSubscription(t *testing.T) string {
+	pubsubSubscription := os.Getenv("SCC_PUBSUB_SUBSCRIPTION")
+	if pubsubSubscription == "" {
+		t.Skip("SCC_PUBSUB_SUBSCRIPTION not set")
+	}
+	return pubsubSubscription
+}
+
+func addNotificationConfig(t *testing.T, notificationConfigID string) error {
```

Comment:
this looks nearly identical to the createNotification sample. can we reuse that in the test to avoid duplicate code?

Reply:
Addressed